File: modul-04/python/PRAK404-2310817310003-Mfaridsetiawan.py
while True:
    print("Pilih program\n1. Penjumlahan\n2. Pengurangan\n3. Perkalian\n4. Pembagian\n5. Exit")
    pilihan = float(input("Masukkan Pilihan: "))

    if pilihan == 1 or pilihan == 2 or pilihan == 3 or pilihan == 4:
        nilai_pertama = float(input("Masukkan Nilai Pertama: "))
        nilai_kedua = float(input("Masukkan Nilai Kedua: "))

        if pilihan == 1:
            hasil = nilai_pertama + nilai_kedua
            print(f"Hasil penjumlahan antara {nilai_pertama:.2f} dengan {nilai_kedua:.2f} adalah {hasil:.2f}\n")
        elif pilihan == 2:
            hasil = nilai_pertama - nilai_kedua
            print(f"Hasil pengurangan antara {nilai_pertama:.2f} dengan {nilai_kedua:.2f} adalah {hasil:.2f}\n")
        elif pilihan == 3:
            hasil = nilai_pertama * nilai_kedua
            print(f"Hasil perkalian antara {nilai_pertama:.2f} dengan {nilai_kedua:.2f} adalah {hasil:.2f}\n")
        elif pilihan == 4:
            hasil = nilai_pertama / nilai_kedua
            print(f"Hasil pembagian antara {nilai_pertama:.2f} dengan {nilai_kedua:.2f} adalah {hasil:.2f}\n")
    elif pilihan == 5:
        print("Terimakasih, telah menggunakan kalkulator M.Farid Setiawan")
        break
    else:
        print("Input Anda salah, silahkan coba lagi\n\n")

File: modul-04/bahasa c/PRAK404-2310817310003-Mfaridsetiawan.c
#include <stdio.h>

int main(void)
{

    while (1)
    {
        printf("Pilih program\n1. Penjumlahan\n2. Pengurangan\n3. Perkalian\n4. Pembagian\n5. Exit\nMasukkan Pilihan : ");
        float pilihan, nilaiPertama, nilaiKedua, hasil;
        scanf("%f", &pilihan);
        if (pilihan == 1 || pilihan == 2 || pilihan == 3 || pilihan == 4)
        {
            printf("Masukkan Nilai Pertama : ");
            scanf("%f", &nilaiPertama);
            printf("Masukkan Nilai Kedua : ");
            scanf("%f", &nilaiKedua);

            if (pilihan == 1)
            {
                hasil = nilaiPertama + nilaiKedua;
                printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f\n\n", nilaiPertama, nilaiKedua, hasil);
            }
            else if (pilihan == 2)
            {
                hasil = nilaiPertama - nilaiKedua;
                printf("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f\n\n", nilaiPertama, nilaiKedua, hasil);
            }
            else if (pilihan == 3)
            {
                hasil = nilaiPertama * nilaiKedua;
                printf("Hasil perkalian antara %.2f dengan %.2f adalah %.2f\n\n", nilaiPertama, nilaiKedua, hasil);
            }
            else if (pilihan == 4)
            {
                hasil = nilaiPertama / nilaiKedua;
                printf("Hasil pembagian antara %.2f dengan %.2f adalah %.2f\n\n", nilaiPertama, nilaiKedua, hasil);
            }
        }
        else if (pilihan == 5)
        {
            printf("Terimakasih, telah menggunakan kalkulator M.Farid Setiawan");
            break;
        }
        else
        {
            printf("Input anda salah, silahkan coba lagi\n\n");
        }
    }
}